schema/release-schema.json

```
@@ -808,7 +808,7 @@
         },
         "value": {
           "title": "Value",
-          "description": "The value of this contract. A negative value indicates that the contract will involve payments from the supplier to the buyer (commonly used in concession contracts). This field should not contain the value of a framework agreement; those should be given in estimatedValue or maximumValue instead.",
+          "description": "The value of this contract. A negative value indicates that the contract will involve payments from the supplier to the buyer (commonly used in concession contracts). The value may be updated during implementation (for example, to reflect an amendment). This field should not contain the value of a framework agreement; those should be given in estimatedValue or maximumValue instead.",
```

```suggestion
          "description": "The value of the contract. A negative value indicates that the contract will involve payments from the supplier to the buyer (commonly used in concession contracts). The value may be updated during implementation (for example, to reflect an amendment). This field should not contain the value of a framework agreement; those should be given in estimatedValue or maximumValue instead.",
```
Discussed in https://docs.google.com/spreadsheets/d/1WaN5cdU5UxoCMkq8Od5zlLj68-ckn1lnzWpt_DfzqHk/edit#gid=0 how most fields use the pattern "The <object> of the <object>" rather than "The <object> of this <object>"